packages/jaeger-ui/src/model/transform-trace-data.tsx

```diff
@@ -15,10 +15,11 @@
 import _isEqual from 'lodash/isEqual';
 
 import { getTraceSpanIdsAsTree } from '../selectors/trace';
+import { getConfigValue } from '../utils/config/get-config';
 import { KeyValuePair, Span, SpanData, Trace, TraceData } from '../types/trace';
 import TreeNode from '../utils/TreeNode';
 
-function deduplicateTags(spanTags: Array<KeyValuePair>) {
+export function deduplicateTags(spanTags: Array<KeyValuePair>) {
```

Comment:
I believe this is only exported for tests, so please add `// exported for tests` above this line.